### PR TITLE
Update webostv with more real-life example of turn on automation

### PR DESCRIPTION
**Pull request by igrybkov**
## Proposed change

The current example of the turn-on automation is broken and not working. I'm providing the one that actually works and could be easily copy-pasted with a change of identifiers

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by thecode** [comment]
I think it is good to add the `alias` to the example, but in my opinion `device_id` is mostly for automations from the UI and not using `YAML`, when using `YAML` most users will probably use `entity_id` as the trigger.

File: source/_integrations/webostv.markdown [hunk 1]
```diff
@@ -49,14 +49,17 @@ Automations can also be created using an automation action:
...
-    - platform: webostv.turn_on
-      entity_id: media_player.lg_webos_smart_tv
-  action:
-    - service: wake_on_lan.send_magic_packet
-      data:
-        broadcast_address: 192.168.1.255
-        mac: AA-BB-CC-DD-EE-FF
+  - alias: Turn On Living Room TV with WakeOnLan
+    trigger:
+      - platform: device
```
Comment (thecode) [inline source/_integrations/webostv.markdown]: Added @frenck for an additional opinion since I have less experience in this

Comment (frenck) [reply source/_integrations/webostv.markdown]: I'm not understanding why we should use device automation here?
in general, device automation are less friendly in YAML use case.

Could you explain what was wrong with the existing automation @igrybkov?

**Comment by igrybkov** [comment]
@frenck, well actually nothing is wrong, that's my mistake here. I think I've seen Home Assistant complaining about incorrect platform, and that's why I created the PR. But it's probably was something wrong with my environment or I just trusted vscode plugin validation without checking — either of these. I've just double checked the automation as it is and it works well with `webostv.turn_on` as a platform. Sorry about that!

I've rolled back my changes regarding the trigger, but since PR and we all already here, per @thecode's suggestion, I've updated PR to add alias to the automation and also updated Mac address example to match the actual Mac address format because that's how it's used in [`wake_on_lan` documentation](https://www.home-assistant.io/integrations/wake_on_lan/).
These are cosmetic changes now without actual contribution, so feel free to close the PR if you want.

**Review by thecode — DISMISSED** [review]
Thanks @igrybkov 👍

File: source/_integrations/webostv.markdown [hunk 2]
```diff
@@ -49,13 +49,14 @@ Automations can also be created using an automation action:
...
 
 automation:
-  trigger:
-    - platform: webostv.turn_on
-      entity_id: media_player.lg_webos_smart_tv
-  action:
-    - service: wake_on_lan.send_magic_packet
-      data:
-        mac: AA-BB-CC-DD-EE-FF
+  - alias: Turn On Living Room TV with WakeOnLan
```
Comment (frenck) [inline source/_integrations/webostv.markdown]:
```suggestion
  - alias: "Turn On Living Room TV with WakeOnLan"
```

**Review by thecode — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks @igrybkov 👍

**Pull request merged**